pennylane/spin/spin_hamiltonian.py

```diff
@@ -109,12 +111,12 @@ def heisenberg(lattice, n_cells, coupling=None, boundary_condition=False, neighb
     where ``J`` is the coupling constant defined for the Hamiltonian, and ``i,j`` represent the indices for neighbouring spins.
 
     Args:
-       lattice (str): Shape of the lattice. Input Values can be ``'chain'``, ``'square'``, ``'rectangle'``,
+       lattice (str): Shape of the lattice. Input values can be ``'chain'``, ``'square'``, ``'rectangle'``,
                    ``'honeycomb'``, ``'triangle'``, or ``'kagome'``.
        n_cells (List[int]): Number of cells in each direction of the grid.
        coupling (List[List[float]] or List[math.array[float]]): Coupling between spins, it can be a 2D array
-                    of shape (neighbour_order, 3) or a 3D array of shape 3 * number of spins * number of spins.
-                    Default value is [1.0, 1.0, 1.0].
+                    of shape (``neighbour_order``, 3) or a 3D array of shape :math:`3 \times` ``number of spins``
+                    :math:`\times` ``number of spins``. Default value is [1.0, 1.0, 1.0].
```

Comment:
For the 3D array, better to use "," instead of "\times" and put the shape in parenthesis for consistency.